static/js/settings_account.js

```diff
@@ -104,6 +104,25 @@ exports.set_up = function () {
         }
     });
 
+    $('#forgot_password').on('click', function () {
+        if ($('#forgot_password').html() === 'Forgotten it?') {
```

Comment:
This won't work if the user is using a language other than English -- the text will be "Passwort vergessen?" or something else.

Instead of changing the text of the element, let's have an element for every step and just have all but one of them set to `display: none`. Then use `show` and `hide` to manipulate the state -- see examples in this file and elsewhere in the codebase.